Library/Homebrew/test/dev-cmd/bottle_spec.rb

```diff
@@ -40,3 +41,67 @@
     end
   end
 end
+
+describe Homebrew do
+  subject(:homebrew) { described_class }
+
+  let(:hello_big_sur) { TEST_FIXTURE_DIR/"bottle-merge/hello_big_sur.json" }
+  let(:hello_catalina) { TEST_FIXTURE_DIR/"bottle-merge/hello_catalina.json" }
+  let(:unzip_big_sur) { TEST_FIXTURE_DIR/"bottle-merge/unzip_big_sur.json" }
+  let(:unzip_catalina) { TEST_FIXTURE_DIR/"bottle-merge/unzip_catalina.json" }
+
+  specify "::merge_jsons" do
+    expect(
+      homebrew.merge_jsons(
+        homebrew.read_jsons([hello_big_sur, hello_catalina, unzip_big_sur, unzip_catalina]),
+      ),
+    ).to eq(
```

Comment:
I don't think you need to test equality here because you're effectively writing a test for the JSON logic. Instead, just check they aren't empty and are all parsed.